core/ts/lib/cryptoUtils.ts

```diff
@@ -0,0 +1,143 @@
+import * as assert from "assert";
```

Comment:
In general, other folders have "utils" saved inside an util folder or placed directly inside the ts folder, but if you feel strongly about calling it "lib", we should make sure to maintain consistency across all packages. I personally have no preference, just would be great to see all follow the same standard

Reply:
Naming is always interesting topic but I'll just follow the current convetion for now